src/routes/tools/_common/helpers.js

```diff
@@ -234,3 +234,26 @@ export const convertAnnualRateToSum = ({ date, value }) => {
     return value * 365;
   }
 };
+
+/**
+ * Create initial configuration
+ * @param {object} defaultParams - default params for tool
+ * @param {object} urlParams - params generated from url query string by sapper
+ * @return {object}
+ */
+// Helper function to create an object with initial settings for a Cal-Adapt tool
+export const getInitialConfig = (defaultParams = {}, urlParams = {}) => {
```

Comment:
I might reverse these function params so that `urlParams` is first and `defaultParams` is second. Then you could have `defaultParams` default to the `INITIAL_CONFIG` common const and might not need to import it in the tool. That would make using this function a little simpler.

Reply:
@clhenrick I tried that first and then left it here because sometimes we want to start with a different `INITIAL_CONFIG`. See the update for the Degree Days tool in https://github.com/berkeley-gif/caladapt-website-2021/pull/179/commits/0fe226317029d5b3f5c9620779f1ab285df22111. But will think about this a bit more.

Reply:
But you could still override it with the function params in the opposite order right? Not following why that doesn't work.

Reply:
I see what you're saying. Never mind. Switched the param order in https://github.com/berkeley-gif/caladapt-website-2021/pull/179/commits/0529d0355f4066fd8772dab449ab63b3da48bc04.